Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2983,7 +2983,7 @@ static EbErrorType init_svt_vp9_encoder_handle(
 #endif
 
 #ifndef _RSIZE_T_DEFINED
-typedef size_t Rsize;
+typedef size_t rsize_t;
```

Comment:
from AV1?

Reply:
already defined in Ebdefinition.h?

Reply:
Rsize was seeming to give error when compiling with mingw, using rsize_t and replacing Rsize worked fine

error:
``` none
In file included from ../Source/Lib/VPX/vpx_dsp_rtcd.h:33,
                 from ../Source/Lib/VPX/loopfilter.c:13:
../Source/Lib/Codec/EbDefinitions.h:423:8: error: unknown type name 'Rsize'
  423 | extern Rsize
      |        ^~~~~
../Source/Lib/Codec/EbDefinitions.h:424:27: error: unknown type name 'Rsize'; did you mean 'rsize_t'?
  424 | strnlen_ss(const char *s, Rsize smax);
      |                           ^~~~~
      |                           rsize_t
```